client/components/gravatar/index.jsx

```diff
@@ -1,37 +1,46 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component } from 'react';
 import url from 'url';
 import qs from 'querystring';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import safeImageURL from 'lib/safe-image-url';
+import {
+	getUserTempGravatar
+} from 'state/current-user/gravatar-status/selectors';
 
-module.exports = React.createClass( {
-	displayName: 'Gravatar',
+export class Gravatar extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
```

Comment:
Instead of having to write constructor just to set the initial state, I discovered that we can use ES7 static properties to define the initial state as well. This is how:

```js
export class Gravatar extends Component {
   state = {
      failedToLoad: false
   };

 // ... other code follows
```

Note: I'm not 100% sure it'll work, actually. Haven't tried myself. :)